web/angular-wallet/src/app/main/send-burst/send-multi-out-form/send-multi-out-form.component.ts

```diff
@@ -86,23 +86,39 @@ export class SendMultiOutFormComponent extends UnsubscribeOnDestroy implements O
     }
   }
 
+  private async sendBurstSameAmount(): Promise<void> {
+    const request = {
+      recipientIds: this.recipients.map(r => convertAddressToNumericId(r.addressRS)),
+      fee: convertNumberToNQTString(parseFloat(this.fee)),
+      amount: convertNumberToNQTString(parseFloat(this.amount)),
+      pin: this.pin,
+      keys: this.account.keys,
+    };
+    await this.transactionService.sendSameBurstToMultipleRecipients(request);
+  }
+
+  private async sendBurstArbitraryAmount(): Promise<void> {
+    const request = {
+      recipientAmounts: this.recipients.map(r => ({
+        recipient: convertAddressToNumericId(r.addressRS),
+        amountNQT: convertNumberToNQTString(parseFloat(r.amount))
```

Comment:
seems we are using `amountNQT` to represent NQT here, but on line 93 we are using `amount` which is also in NQT. Should `amount` on line 93 be in NQT?

Reply:
It's a naming issue. For sake of consistency I'll rename `amount` of `SendBurstMultipleSameRequest` in `amountNQT`

Reply:
done